budgea_openapi.json

```diff
@@ -21224,49 +21224,49 @@
         "parameters": [
           {
             "required": false,
-            "in": "formData",
```

Comment:
Could please re-explain briefly the rationale behind this change `formData` => `query`?

Reply:
The method is `get` so it makes no sense to send formData 🤷‍♂